internal/app/cli/app.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+//
+// This file is part of Storj CLI.
+//
+// Storj CLI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// Storj CLI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with Storj CLI.  If not, see <http://www.gnu.org/licenses/>.
+
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+	storj "storj.io/storj/pkg/client"
```

Comment:
I decided this way, because it is this:
```go
storj.GetBuckets(storj.NewEnv())
```
looks more self-explanatory than this:
```go
client.GetBuckets(client.NewEnv())
```

Reply:
that's fair, but this is pretty surprising from a go convention perspective. there are a number of go tools (goimports, for instance) that will automatically find your imports for you based on how you use them, and most tooling expects unnamed imports. named imports should be a rare exception.